nc-news/src/api.jsx

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios'
+
+export const api = (url, type='get', body) => {
+    if (type === 'get') {
```

Comment:
We don't need conditional logic in here - please look at the axios docs to see how to better formulate this page. You're also re-writing this function in your Articles.jsx file, again, if it's not useful for this ticket it should be taken out.